mula/scheduler/schedulers/boefje.py

```diff
@@ -997,3 +997,29 @@ def get_boefjes_for_ooi(self, ooi) -> list[Plugin]:
         )
 
         return boefjes
+
+    def set_cron(self, item: Task) -> str | None:
```

Comment:
Given the complexity and the implementation below, I believe using this as a setter instead of property is a good choice 👍